### PR TITLE
Small typo in README

### PR DESCRIPTION
**Pull request by rajasec**
Fixing small typo in README file
Signed-off-by: rajasec <rajasec79@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ runc run mycontainerid
 If you used the unmodified `runc spec` template this should give you a `sh` session inside the container.
 
 The second way to start a container is using the specs lifecycle operations.
-This gives you move power of how the container is created and managed while it is running.
+This gives you more power of how the container is created and managed while it is running.
```
Comment (cyphar) [inline README.md]: `of` is also wrong, it should read:

> This gives you **more** power **over** how the container is created and managed while it is running.

Comment (rajasec) [inline README.md]: @cyphar 
Updated.

**Comment by rajasec** [comment]
@cyphar 
Updated.

**Comment by mrunalp** [comment]
LGTM

**Comment by cyphar** [comment]
LGTM

**Pull request merged**